edb/server/pgrust/Cargo.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+name = "pgrust"
+version = "0.1.0"
+license = "MIT/Apache-2.0"
+authors = ["MagicStack Inc. <hello@magic.io>"]
+edition = "2021"
+
+[lint]
+workspace = true
+
+[features]
+python_extension = ["pyo3/extension-module", "pyo3/serde"]
+optimizer = []
+
+[dependencies]
```

Comment:
There are more dependencies here than necessary but we'll land the rest when #7605 lands